docs/recipes/build/rollup.md

```diff
@@ -0,0 +1,135 @@
+# [Rollup](https://github.com/rollup/rollup) Bundling
+
+## Dependencies
+
+- A working installation of `node` (4.0+) & `npm`.
+- `npm install --save-dev sw-build sw-lib rollup rollup-plugin-node-resolve`
+
+## Service Worker Code
+
+This approach assumes that your `src/sw.js` file is using [ES2015 module
+import](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/import)
+syntax, and is structured like:
+
+```js
+// Contents of src/sw.js:
+
+import manifest from '/tmp/manifest.js';
+import swLib from 'sw-lib';
+// import other libraries, e.g. idb-keyval, Firebase Messaging, etc.
+
+swLib.cacheRevisionedAssets(manifest);
+
+// Additional runtime service worker logic goes here.
+```
+
+## Option 1: Vanilla node Build Script
+
+The following is a basic `node` build script that can be run as part of a larger
```

Comment:
I don't understand this paragraph - what is the goal?

Why is it referencing npm scripts?